e2e-tests/src/accounts.rs

```diff
@@ -27,3 +31,34 @@ pub fn accounts_seeds_to_keys(seeds: &[String]) -> Vec<KeyPair> {
 pub fn get_sudo_key(config: &Config) -> KeyPair {
     keypair_from_string(&config.sudo_seed)
 }
+
+pub struct NodeKeys {
+    pub validator_key: KeyPair,
+    pub controller_key: KeyPair,
+    pub stash_key: KeyPair,
+}
+
+impl From<u32> for NodeKeys {
+    fn from(seed: u32) -> Self {
+        let validator_seed = get_validator_seed(seed);
+        NodeKeys::from(validator_seed)
+    }
+}
+
+impl From<String> for NodeKeys {
+    fn from(seed: String) -> Self {
+        Self {
+            validator_key: keypair_from_string(&seed[..]),
+            controller_key: keypair_from_string(&get_validators_controller_seed(seed.clone())[..]),
+            stash_key: keypair_from_string(&get_validators_stash_seed(seed.clone())[..]),
+        }
+    }
+}
+
+fn get_validators_controller_seed(seed: String) -> String {
+    format!("{}//Controller", seed)
+}
+
+fn get_validators_stash_seed(seed: String) -> String {
+    format!("{}//stash", seed)
```

Comment:
`  format!("{}//Stash", seed)`? Your version might be correct, but there seems to be an inconsistency in using `{}//Stash` vs. `{}//stash` in various parts of the repo.